tests/namespace/test_fdb.py

```diff
@@ -23,6 +23,7 @@
 class TestSonicMIB(TestCase):
     @classmethod
     def setUpClass(cls):
+        cls.skipTest(cls, "Namespace not implemented")
```

Comment:
>skipTest [](start = 12, length = 8)

What is the meaning of "Namespace not implemented"?
Why you skip test?

Reply:
For multi-asic platform, the change is made to if_id_map key to store namespace and sai id together.
with this change, MIBS using this key have to be updated to use the new type of key.
Instead of making this change on all MIBS together in same PR, will make change in other PRs.
So, for multi-asic platform, the unit-tests for these MIB implementations are skipped.

Reply:
@SuvarnaMeenakshi better we should add similar comment in test case itself  as TODO so that is easy to understand and track

Reply:
Updated comment with TODO